base_view_inheritance_extension/models/res_users.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.multi
+    def unlink(self):
+        """
+        Finds any views assigned specifically to this user and deletes them.
+        """
+        self.env['ir.ui.view'].sudo().with_context(
```

Comment:
this can be more readable by doing

```python
codecodecode.with_context(
    key=value
).search([
    propositions
]).filtered(
    lambda
).unlink()